app/src/main/java/campus/tech/kakao/map/presentation/viewmodel/SplashScreenViewModel.kt

```diff
@@ -0,0 +1,37 @@
+package campus.tech.kakao.map.presentation.viewmodel
+
+import android.util.Log
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.google.firebase.ktx.Firebase
+import com.google.firebase.remoteconfig.FirebaseRemoteConfig
+import com.google.firebase.remoteconfig.ktx.remoteConfig
+import com.google.firebase.remoteconfig.ktx.remoteConfigSettings
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+
+@HiltViewModel
+class SplashScreenViewModel @Inject constructor(
+    private val remoteConfig: FirebaseRemoteConfig
+) : ViewModel() {
+
+    private val _serviceState = MutableLiveData<String>()
+    val serviceState: LiveData<String> get() = _serviceState
+
+    private val _serviceMessage = MutableLiveData<String>()
+    val serviceMessage: LiveData<String> get() = _serviceMessage
+
+    init {
+        fetchFirebaseValues()
+    }
+
+    private fun fetchFirebaseValues() {
+        remoteConfig.fetchAndActivate().addOnCompleteListener {
+            if (it.isSuccessful) {
+                _serviceState.value = remoteConfig.getString("serviceState")
+                _serviceMessage.value = remoteConfig.getString("serviceMessage")
```

Comment:
지금은 RemoteConfig에서 받는 값이 많지 않지만, 나중에 기능을 붙이다보면 점점 많아질 수도 있을 거예요. 그럴 때는 지금처럼 config값마다 LiveData를 연결하기엔 다소 비효율적일 수도 있겠어요.
config값의 개수에 따라 전체값을 필드로 갖는 data class를 하나 만들 수도 있고, 더 늘어난다면 그에 맞게 클래스도 여러개로 나눠서 세분화해볼 수도 있을 것 같아요. 

```suggestion
data class RemoteConfigInfo(val state: String, val message: String)
// ...
_remoteConfigInfo.value = RemoteConfigInfo(
    state = remoteConfig.getString("serviceState")
    message = remoteConfig.getString("serviceMessage")
)
```